### DISCUSSION TITLE
Something between this and the MVP

### DISCUSSION THREAD

**Discussion by thedevdavid** [colbyfayock/next-wordpress-starter#231] [Ideas]
Hi Colby,

It'd be great to have a version that is somewhere in between this and the MVP. To have access to all of the configuration, and WP API stuff (like pages, menus, users, etc.) but without the opinionated styling and usage of more NextJs ecosystem (like next-seo).

Right now it's a lot of work to set up a new website with a custom design using this starter, but if I use the MVP starter, it's just as much work to get the pages, search, etc. included.

**Comment by colbyfayock** [comment]
hey @DLevai94 thanks for the feedback. i dont have any plans to maintain a separate starter but youre welcome to fork this and create something like that!
